src/client/src/MessageList.jsx

```diff
@@ -0,0 +1,20 @@
+import './styles/style.css'; 
+import moment from 'moment';
+
+export const MessageList = ({ messages }) => {
+  return (
+    <ul>
+      {messages.map(message => {
+        const dateString = message.createdAt;
```

Comment:
The variable name 'dateString' may not be descriptive enough. Consider renaming it to 'messageCreatedAt' to better match its contents.